talkativeRobotRGB.rb

```diff
@@ -1,8 +1,29 @@
-puts("How old are you?")
-age = gets().chomp().to_i()
+def askAge
+  puts("How old are you?")
+  gets().chomp().to_i()
+end
+
+def askGender
+  puts("Are you male or female (M/F)?")
+  gets().chomp()
+end
 
-puts("Are you male or female (M/F)?")
-genderInput = gets().chomp()
+def lastMessage(ageDifference,gender)
+
+  if (ageDifference > 60 && gender == "M")
```

Comment:
Don't need the `()` around any of the `if` or `elsif` statements. It adds extra clutter.